apps/projects/app/components/Content/Issues.js

```diff
@@ -473,39 +534,45 @@ class Issues extends React.PureComponent {
                 repoId => downloadedRepos[repoId].hasNextPage
               ).length > 0
 
+            const issuesSorted = this.shapeIssues(issuesFiltered)
+              .sort(currentSorter)
+
+            const estimatedHeight = DEFAULT_CARD_HEIGHT * issuesSorted.length
+
+            const onScroll = ({ scrollTop }) => {
+              const position = scrollTop / estimatedHeight
+              if(moreIssuesToShow && position >= LOAD_MORE_THRESHOLD) {
+                this.showMoreIssues(downloadedIssues, downloadedRepos)
+              }
+            }
+
             return (
               <StyledIssues>
                 {this.actionsMenu(downloadedIssues, issuesFiltered)}
                 {this.filterBar(downloadedIssues, issuesFiltered)}
 
                 <IssuesScrollView>
-                  <ScrollWrapper>
-                    {this.shapeIssues(issuesFiltered)
-                      .sort(currentSorter)
-                      .map(issue => (
-                        <Issue
-                          isSelected={issue.id in this.state.selectedIssues}
-                          key={issue.id}
-                          {...issue}
-                          onClick={this.handleIssueClick}
-                          onSelect={this.handleIssueSelection}
-                        />
-                      ))}
-                  </ScrollWrapper>
-
-                  <div style={{ textAlign: 'center' }}>
-                    {moreIssuesToShow && (
-                      <Button
-                        style={{ margin: '12px 0 30px 0' }}
-                        mode="secondary"
-                        onClick={() =>
-                          this.showMoreIssues(downloadedIssues, downloadedRepos)
-                        }
-                      >
-                        Show More
-                      </Button>
+
+                  <AutoSizer
+                    defaultHeight={window.innerHeight}
+                    disableWidth={true}
+                  >
+                    {({ height }) => (
+                      <List
+                        autoWidth={true}
```

Comment:
Good question. When I set `width` to `window.innerWidth` without setting an `autoWidth` prop, the width of the list expands a little too much. When I set `autoWidth` without `width`, I get an error because `width` is a required prop. So really `autoWidth` is just a little hack that I found by trial and error (which, by the way, *is* a valid prop for `Grid`, which `List` uses internally).